EngineTest/Source/Core/Intersectable/SIMD_triangle_intersection_test.cpp

```diff
@@ -18,6 +18,9 @@
 #error "Unsupported platform"
 #endif
 
+
+ph::testTriangle normaltri[1000000];
+ph::PackedTriangle packedtri[1000000];
```

Comment:
note that Travis VM has quite a strict memory limit, these sizes may cause the unit test to fail